system/nxplayer/nxplayer_offload.c

```diff
@@ -0,0 +1,321 @@
+/****************************************************************************
+ * apps/system/nxplayer/nxplayer_offload.c
```

Comment:
should we name the file nxplayer_offload_mp3.c?

Reply:
done